### PR TITLE
style: Match design system

### LINKED ISSUE #2113: Create a Design System and apply it to the current UI
**Describe the solution you'd like**

Incrementally rollout the new design system for app.flagsmith.com.

## Stage 0 - Setup

- [x] Upgrade Bootstrap, setup bootstrap variables with the aim on removing custom css for components (e.g. button-primary) https://github.com/Flagsmith/flagsmith/pull/2289

## Stage 1 - Components 

Implement components, utilising bootstrap utils (ms-5 col-md etc text-<color> bg-<color>) etc and deleting bespoke css.

If a component has a variant, implement a theme property (See Button.tsx) that allows defining variants as properties.

All components should be tested in both light and dark mode and in Safari/Chrome.

- [x] Create button component with all variants
- [x] Typography (size, family, weight and line heights)
- [x] Text input / area
- [x] Switch
- [x] Dark mode switcher
- [x] Checkbox
- [x] Modals (generic, not styling every individual modal's content)
- [x] Info
- [x] Error message
- [x] Select
- [x] Tags
- [x] Chips (e.g. invite users)
- [x] Tabs (full width) + Pills - combine these to 1 component with a theme "tab" | "pill"
- [x] Radio
- [x] CodeHelp
- [x] Date select
- [x] Side Modals - this will require matching styles of the segment and feature modals
- [x] Panels and list items
Double Check https://github.com/Flagsmith/flagsmith/issues/2463 to make sure it is resolved
- [x] Side nav + navbar / combine projects page with org dropdown
- [x] Identity / Segment override icons
- [x] Panel pagination

## Stage 2 - Page / Organism development 

Implement grid based layouts according to the column view in Figma, these should be responsive and use standard column, padding and margin utils. Avoid bespoke styles, delete any current bespoke styles that we are replacing, avoid adhoc pixel values. 

**Add pages (e.g. login, features) as you go, modals e.g. schedule updates, create/edit feature.**

- [x] Account tabs
- [x] Features List
- [x] Create First Organisation  
- [x] Project Settings
- [x] Compare
- [x] Segments List
- [x] Create / Edit  Segment 
- [x] Login / Signup
- [x] Segment overrides
- [x] Create First Project
- [x] Project list
- [x] Edit Feature
- [x] Feature-Specific segments 
- [x] Feature Settings
- [x] Audit Log
- [x] Integrations
- [x] Scheduling 
- [x] Create environment 
- [x] Change Request list
- [x] Change Request detail 
- [x] Identities
- [x] Identity  
- [x] Environment Settings   
- [x] Organisation settings 
- [x] Dark mode run-through

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes
Allows us to close the [Design system PR](https://github.com/Flagsmith/flagsmith/issues/2113). Runs through all the pages listed and applies any notable design errors.

## How did you test this code?

- Ran through each of the pages in production

### CODE REVIEW

File: frontend/web/components/InfoMessage.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default class InfoMessage extends PureComponent {
     return (
       <div className={infoMessageClass}>
         <span className={`icon-alert ${this.props.infoMessageClass} info-icon`}>
-          <Icon name='info' />
+          <Icon fill={'#0AADDF'} name={this.props.icon || 'info'} />
```
Comment (kyle-ssg) [inline frontend/web/components/InfoMessage.js:25]: image

File: frontend/web/components/PanelSearch.js [hunk 2]
```diff
@@ -212,7 +212,7 @@ const PanelSearch = class extends Component {
               {!!this.props.filterRow && (
                 <Row>
                   {this.props.showExactFilter && (
-                    <div style={{ width: 140 }}>
+                    <div className='me-3' style={{ width: 140 }}>
```
Comment (kyle-ssg) [inline frontend/web/components/PanelSearch.js:215]: image

File: frontend/web/components/PanelSearch.js [hunk 3]
```diff
@@ -329,7 +329,9 @@ const PanelSearch = class extends Component {
             />
           )}
           {this.props.renderFooter && (
-            <div className='panel-footer'>{this.props.renderFooter()}</div>
+            <footer className='panel-footer'>
```
Comment (kyle-ssg) [inline frontend/web/components/PanelSearch.js:332]: Border radius was incorrect and removed the border

image



image

File: frontend/web/components/SegmentOverrides.js [hunk 4]
```diff
@@ -593,7 +591,7 @@ class TheComponent extends Component {
                   !this.props.disableCreate && (
                     <div className='text-right'>
                       <Button
-                        className='mt-2'
+                        className='mt-4'
```
Comment (kyle-ssg) [inline frontend/web/components/SegmentOverrides.js:594]: image

File: frontend/web/components/SegmentOverrides.js [hunk 5]
```diff
@@ -634,6 +632,7 @@ class TheComponent extends Component {
                 )}
                 {this.props.showCreateSegment && this.state.segmentEditId && (
                   <CreateSegmentModal
+                    className='my-2'
```
Comment (kyle-ssg) [inline frontend/web/components/SegmentOverrides.js:635]: Before : 

image


After:

image

File: frontend/web/components/pages/ChangeRequestsPage.js [hunk 6]
```diff
@@ -200,10 +201,11 @@ const ChangeRequestsPage = class extends Component {
                               )}
                             </div>
                             <div className='list-item-subtitle mt-1'>
-                              Created at{' '}
+                              Created{' '}
                               {moment(created_at).format('Do MMM YYYY HH:mma')}{' '}
                               by {user && user.first_name}{' '}
                               {user && user.last_name}
+                              {description ? ` - ${description}` : ''}
```
Comment (kyle-ssg) [inline frontend/web/components/pages/ChangeRequestsPage.js:208]: image

File: frontend/web/components/SegmentOverrides.js [hunk 7]
```diff
@@ -175,48 +175,46 @@ const SegmentOverrideInner = class Override extends React.Component {
...
-                                <Icon name='edit' fill={'#656D7B'} width={20} />
-                              </span>
-                            </Button>
-                          )}
-                        </>,
-                      )
-                    }
-                  </Permission>
-                )}
+                <Permission
```
Comment (kyle-ssg) [inline frontend/web/components/SegmentOverrides.js:178]: Prior to this you couldn't go back and edit a newly created feature-based segment

image


image

**Review by novakzaballa — APPROVED** [review]
LGTM. Thank you for attaching the screenshots

**Pull request merged**